ivc/tests/add.rs

```diff
@@ -350,6 +355,147 @@ pub fn heavy_test_simple_add() {
         (),
         GenericVecStructure<Curve>,
     >;
+    type MainTestConfig = Config<N_COL_TOTAL, N_FSEL_TOTAL>;
+
+    ////////////////////////////////////////////////////////////////////////////
+    // Evaluations
+    ////////////////////////////////////////////////////////////////////////////
+
+    #[allow(dead_code)]
```

Comment:
I would guess it is used later.

Reply:
Is removed eventually as far as I remember.